austinguitest.py

```diff
@@ -265,19 +282,15 @@ def gameplay(self):
         #if there is some sort of userInput put in, check if it's yes then ask for fileName
         #run save function, else in the end it will generate a new puzzle.
         if(self.controller.controllerGetPuzzleState() == 1):
-            userinput = simpledialog.askstring("Would you like to save your game?",  "Would you like to save? yes/no:")
-            if userinput:
-                if userinput.lower() == "yes":
+            answer = messagebox.askyesno("Would you like to save?", "Would you like to save the game?")
+            if answer == True:
                     fileName = simpledialog.askstring("Please enter a file name", "File name: ")
                     self.controller.controllerSaveGame(fileName)
-                elif userinput.lower() == "no":
+            elif answer == False:
                     messagebox.showinfo("Generating a new puzzle", "Time to generate a new puzzle!")
-                else:
-                    messagebox.showinfo("Error", "Enter yes/no:")
-                    return
             else:
-                 messagebox.showinfo("No information provided", "Cancelled, try again")
-                 return
+                messagebox.showinfo("Error", "Choose yes or no!")
```

Comment:
Wouldn't this be if they clicked no? Why would it give an error and prompt again?